test/Java9andUp/src/org/openj9/test/java9AttachAPI/SelfAttacher.java

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
```

Comment:
Please update all the licenses to be the lastest version.

Reply:
It appears the middle part of the licenses was updated, but not the ending part.